application/Modules/Clipboard/Producers/ClipboardCopyEventProducer.cs

```diff
@@ -20,11 +20,12 @@ public class ClipboardCopyEventProducer : DefaultEventQueue<ClipboardCopyEvent>
         ///     threw an exception.
         ///!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 
         /// </summary>
-        private static INativeClipboard nativeClipboard = new NativeClipboard();
 
         private static readonly ClipboardWindowMessageSink
             clipboardWindowMessageSink = new ClipboardWindowMessageSink();
 
+        private INativeClipboard nativeClipboard = clipboardWindowMessageSink.NativeClipboard;
```

Comment:
This could be made `readonly`
Could you also remove the comment above as it doesn't seem to be applicable anymore?